src/main/java/org/jfrog/hudson/pipeline/common/types/buildInfo/BuildInfo.java

```diff
@@ -63,11 +64,13 @@ public BuildInfo(Run build) {
         this();
         this.name = BuildUniqueIdentifierHelper.getBuildName(build);
         this.number = BuildUniqueIdentifierHelper.getBuildNumber(build);
+        issues.setBuildName(name);
```

Comment:
Please add for consistency:
```suggestion
        this.issues.setBuildName(name);
```